specutils/io/asdf/types.py

```diff
@@ -0,0 +1,15 @@
+from asdf.types import CustomType
+
+
+_specutils_types = set()
+
+
+class SpecvizType(CustomType):
```

Comment:
Why is this called `SpecvizType` and not `SpecutilsType`?

Reply:
Because 🤦‍♂️